### ISSUE TITLE
Blocks and items wont show in gui on mac

### ISSUE THREAD

**Issue by mrPaintMan** [ldtteam/minecolonies#6816]
### Prerequisites

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 
- Im running the mods on a 2019 macbook pro 15", Intel Iris Plus Graphics 645 1536 MB
- Minecraft Version: 1.16.5
- MineColonies Version: 0.14.58-ALPHA
- Structurize Version: 0.13.148
- Related Mods and their Versions: Optifine HD G6 ULTRA

## Expected behavior
pictures of blocks should be visable in gui (e.g. university gui, or builder gui)

The same setup on a windows computer *is working fine*

## Actual behavior
Pictures of blocks or items are invisible in the gui. 

The university shows some items and blocks when the gui is zoomed out, but they disappear when zoomed in.

This problem occurs in both single and multiplayer

<img width="1440" alt="Screenshot 2021-03-26 at 11 35 19" src="https://user-images.githubusercontent.com/33493180/112621146-b4059480-8e29-11eb-927e-f020e7e856fc.png">

<img width="1440" alt="Screenshot 2021-03-26 at 11 35 47" src="https://user-images.githubusercontent.com/33493180/112620978-7e60ab80-8e29-11eb-9acb-f831b8c846c8.png">

<img width="1440" alt="Screenshot 2021-03-26 at 12 02 05" src="https://user-images.githubusercontent.com/33493180/112622279-1f039b00-8e2b-11eb-8eaa-a37492ea13d5.png">

## Steps to reproduce the issue

1. Run the latest moinecolonies on a mac computer
2. Try to build the builders hut for instance
3. Observe wether items and blocks are displayed in the gui or not.

### Logs

- latest.log: [lastest.log](https://github.com/ldtteam/minecolonies/files/6211326/lastest.log)

- crashlog: 

### Notes
Also the tooltip for the different university researches wont show, making it very hard fpr me as a mac user to use the university at all

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Nightenom** [comment]
Try lower gui scale (if it works then this is dupe of ldtteam/Structurize#328), might be also problem of high dpi when using retina and similar displays - not sure if it's possible to somehow adjust dpi values for applications using window manager, also try without optifine - if it works without try resetting optifine settings and find the problematic one

**Comment by mrPaintMan** [comment]
I lowered the gui scale and it worked! Although i needed quite small gui to get the full functionality back. At gui level 5, i still had some issues with tooltips.

I also tried uninstalling Optifine, but the problem was still there.

<img width="1440" alt="Screenshot 2021-03-26 at 12 31 45" src="https://user-images.githubusercontent.com/33493180/112625485-48262a80-8e2f-11eb-91f7-ce8655e29ae5.png">
